src/gleam_community/colour.gleam

```diff
@@ -187,17 +191,15 @@ fn hex_string_to_int(hex_string: String) -> Result(Int, Nil) {
       case total {
         Error(Nil) -> Error(Nil)
         Ok(v) -> {
-          use
-            num
-          <- result.then(case char {
-              "f" -> Ok(15)
-              "e" -> Ok(14)
-              "d" -> Ok(13)
-              "c" -> Ok(12)
-              "b" -> Ok(11)
-              "a" -> Ok(10)
-              _ -> int.parse(char)
-            })
+          use num <- result.then(case char {
+            "f" -> Ok(15)
+            "e" -> Ok(14)
+            "d" -> Ok(13)
+            "c" -> Ok(12)
+            "b" -> Ok(11)
+            "a" -> Ok(10)
```

Comment:
I would have a slight preference for ascending order here but it isn't a huge deal

Reply:
I'll do it, just for you